db/recreate_parts_table.sql

```diff
@@ -2,7 +2,7 @@
 DROP TABLE if exists parts;
 
 CREATE TABLE parts (
-	id INT PRIMARY KEY,
+	id SERIAL PRIMARY KEY,
```

Comment:
Probably won't matter since INSERT will never be used. Parts will always come from fresh imports of parts.csv